src/test/tgen/fixed_duration/verify.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+printf "TGen configuration: network=$1 client=$2\n"
+network=$1
+client=$2
+
+# read speed of server is the effective write speed of client
+# read speed of client is the effective write speed of server
+# tail -1 gets the fastest speed reached
+server_bps_read="$(grep -r --include="server.tgen.*.stdout" "driver-heartbeat" \
+    | cut -d' ' -f8 \
+    | cut -d'[' -f2 \
+    | cut -d',' -f1 \
+    | cut -d'=' -f2 \
+    | sort -n \
+    | tail -1)"
+client_bps_read="$(grep -r --include="client.tgen.*.stdout" "driver-heartbeat" \
+    | cut -d' ' -f8 \
+    | cut -d'[' -f2 \
+    | cut -d',' -f1 \
+    | cut -d'=' -f2 \
+    | sort -n \
+    | tail -1)"
+
+printf "Server bytes per second=${server_bps_read}\n"
+printf "Client bytes per second=${client_bps_read}\n"
+
+# Test passes if tgen application achieves >= 85% of configured speed.
+# We use a 85% threshold to allow for TCP overhead.
+expected=0
+if [[ "${network}" == "1mbit"* ]]; then
+    expected=106250
```

Comment:
nit: maybe leave a comment on this first value with how it's computed? (1Mbit/s = 125000 bytes/s, 125000 bytes/s * 0.85 = 106250 bytes/s)